### DISCUSSION TITLE
Help with Android Studio error

### DISCUSSION THREAD

**Discussion by sjmalon** [community/community#50836] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm simply trying to share a project on github from android studio and I keep getting this error message on android studio: "Push failed remote error: sjmalon/TD_List.gitsjmalon/TD_List10 is not a valid repository name Visit https://support.github.com/ for help". Can anyone help please?

**Comment by Fo00oX** [comment]
Check https://github.com/orgs/community/discussions/28601

else:

make sure you did this right:
1. Go to the "VCS" (Version Control System) menu in the toolbar.
2. Select "Enable Version Control Integration" if you haven't already.
3. Choose "Git" as the version control system and click "OK".
4. f you haven't committed your changes, go to "VCS" > "Commit" and commit your changes.
5. Go to "VCS" > "Git" > "Remotes".
6. In the "Git Remotes" window, you should see a list of remote repositories. Find the one with the incorrect name (sjmalon/TD_List.gitsjmalon/TD_List10), select it and click the "Edit" button (pencil icon).
7. Update the remote repository name by changing the "URL" field to the correct format: "https://github.com/username/repository.git". In your case, it should be "https://github.com/sjmalon/TD_List.git".
8. Click "OK" to save your changes and close the "Git Remotes" window.
9. Now you should be able to push your changes to GitHub. Go to "VCS" > "Git" > "Push" and push your changes.


Else: 
1. Double-check the repository name and make sure it's correct. Check for any typos or misspellings.

2. Make sure you have permission to push to the repository. If the repository belongs to someone else, they may need to grant you permission to push.

3. Check the repository's settings on GitHub to make sure it's set up correctly. Make sure it's not set to read-only or private.

4. Try creating a new repository with a different name and pushing your project to that repository. This will help you determine if the issue is with the repository name or with something else.

Hope it helps

**Reply by sjmalon** [reply]
I finally got it to work. Something odd was going on where I had to drop the end of the github url and just leave the https://github.com part in manage remotes, so at least there's one way for me to do it. Thanks though!

**Comment by MinaEliaa** [comment]
hi

**Reply by Fo00oX** [reply]
Hi, May I halp you somehow?
